### ISSUE TITLE
Super Basics: HTML, CSS, JS...

### ISSUE THREAD

**Issue by Fawst** [r00k/AMA#27]
What should I focus on next? I enjoy front end, but also am interested in back end. Leaning towards Rails.

**Comment by queuebit** [comment]
Here are a few great starting resources:
- http://prework.flatironschool.com/web-development/
- https://github.com/thoughtbot/trail-map
- https://www.railstutorial.org/book

**Comment by r00k** [comment]
The first thing I would focus on is how to ask a good question, because this one is pretty bad.

You haven't given me nearly enough context to be able to help you. Asking a question this vague will never get you a good answer. How could I possibly offer good advice given this?

**Comment by r00k** [comment]
I'm still happy to help you out, but you need to tell me at least the following:
- What have you learned already? What is your skill level in these skills?
- What are your larger goals?

**Comment by Fawst** [comment]
Sorry, I guess I worded that (quite) poorly! Or, I just don't know the right question to ask. I have a basic-to-intermediate understanding of HTML, CSS and JavaScript. I know "enough to be dangerous," in that I've created basic sites. The most recent example is http://www.strategic-alliances.org/page/summit2015sessions (that page has the most complex functionality). I'm still learning more with those, but I want to move past that.

As for larger goals, I want to move past the intermediate. I want to learn the tools that will allow me to make websites that go beyond just being informative. For instance, my organization's hosted alliance management software is not ideal. If I had to point at a specific target, I would say (for now) that I want to learn how to create an interactive website for members that interacts with a database to store member records. I saw your "Refactoring from Good to Great" which is what got me interested in Ruby. Rails seemed like a good/fun place to dive in.

I'm just not sure where to turn next. I started the Rails Tutorial and I love it so far. I just didn't know if I was getting ahead of myself and if there was some "next step" I was missing. Thanks for reading!
